### DISCUSSION TITLE
Welcome to PeakFit Discussions!

### DISCUSSION THREAD

**Discussion by heyjaywilson** [heyjaywilson/peakfit#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samuelsainz** [comment]
hey there 👋 I've been contributing to PeakFit in the last days and I'm very excited for it! This app would be something I would use myself a lot so I'm looking forward to help building it. I'm also looking forward to contribute to the open-source community and continue learning! 

@heyjaywilson thanks Jay for your work and for open sourcing it! 👏

**Reply by heyjaywilson** [reply]
@samuelsainz glad to hear it! I appreciate your contributions and wanting to work on it! Feel free to drop any feature ideas you have in the ideas discussion. I've been putting features there to think about adding as the base features are getting complete.

There's one or two more tickets before I start getting a public testflight going for it and when I do I'll make another announcement here on how to sign up for that!
